app/controllers/katello/api/v2/repository_sets_controller.rb

```diff
@@ -35,7 +35,7 @@ def show
     param :product_id, :number, :required => true, :desc => N_("ID of a product to list repository sets from")
     def available_repositories
       scan_cdn = sync_task(::Actions::Katello::RepositorySet::ScanCdn, @product, @product_content.content.id)
-      repos = scan_cdn.output[:results]
+      repos = scan_cdn.output[:results] || []
```

Comment:
ahhh, i see.  Defaulting to [] would be fine, but we need to make sure that the error is propagated up in that controller action.  Looks like its just completely ignoring it.